editor.planx.uk/src/components/Header/Header.tsx

```diff
@@ -476,6 +473,14 @@ const EditorToolbar: React.FC<{
     setOpen(!open);
   };
 
+  const logout = async () => {
```

Comment:
I think there's no issues with writing `revoked_token` records to production tables - the records just won't get used or checked yet.